source/adminguide/virtual_machines/importing_unmanaging_vms.rst

```diff
@@ -401,6 +409,170 @@ Unmanaging Instance actions
 
    - For the Instance being unmanaged: stopped and destroyed usage events (similar to the generated usage events when expunging an Instance), with types: ‘VM.STOP’ and ‘VM.DESTROY', unless the instance has been already stopped before being unmanaged and in this case only ‘VM.DESTROY' is generated.
 
+Import Instances from External Hosts
+-------------------------
+.. note:: This is currently only available for **KVM** hypervisor.
+
+External Host
+~~~~~~~~~~~~~
+
+An External Host refers to a host that is not managed by CloudStack. The "Import from external host" feature enables the
+importation of instances from these external hosts. Administrators can execute this import process for instances from
+External KVM hosts either through the user interface (UI) or by utilizing the importVm API.
+
+Prerequisites
+~~~~~~~~~~~~~
+- Ensure that the External KVM host is equipped with the installation of libvirt.
+- Allow TCP libvirt connections from Managed KVM hosts within CloudStack to the External host.
+- It is imperative that instances residing on the external host are in a stopped state, as live migration of instances is not supported
+
+listVmsForImport API
+~~~~~~~~~~~~~~~~~~~~
+
+listVmsForImport API, created for functionality within the External KVM environment, serves the purpose of listing all
+instances currently in a stopped state on the designated External KVM host. To access this information programmatically,
+it is imperative to provide the corresponding username and password associated with the external host. These credentials
+serve as essential authentication measures, enabling secure interaction with the External KVM host and facilitating the
+retrieval of detailed insights into the status of instances that are currently halted or stopped.
+
+**Request parameters**:
```

Comment:
@andrijapanicsb instead of adding request  / response parameters here, can redirect to api docs of these apis?

Reply:
yes and no, I don't have strong opinion, I can see that for the original VM ingestion (VMware) we have these parameters, so I'm fine with these left in here. Feel free to reword if you prefer.